src/Components/Web.JS/src/PageTitle.ts

```diff
@@ -5,7 +5,8 @@ export const PageTitle = {
 };
 
 function getAndRemoveExistingTitle(): string | null {
-  const titleElements = document.getElementsByTagName('title');
+  // Other <title> elements may exist outside <head> (e.g., inside <svg> elements) but they aren't page titles
+  const titleElements = document.head ? document.head.getElementsByTagName('title') : [];
```

Comment:
Why not 
```suggestion
  const titleElements = document.querySelectorAll('head > title');
```

Reply:
Fancy. But what if there's an unrelated `<head>` element ??

I know that's super unlikely but it is, in a sense, more strictly accurate to query inside the official `document.head`. Unless you can think of any other advantage to the `head > title` approach, I'd leave it as-is. Also then we don't have to rebuild the `.js` files.

Reply:
Merging now to get on with it, but LMK if you do think this should change because I am happy to follow up on it.